Content.Server/Commands/Physics/TestbedCommand.cs

```diff
@@ -230,5 +232,55 @@ private void CreateCircleStack(MapId mapId)
                 }
             }
         }
+
+        private void CreatePyramid(MapId mapId)
+        {
+            const byte count = 20;
+
+            // Setup ground
+            var entityManager = IoCManager.Resolve<IEntityManager>();
+            var ground = entityManager.SpawnEntity("BlankEntity", new MapCoordinates(0, 0, mapId)).AddComponent<PhysicsComponent>();
+
+            var horizontal = new EdgeShape(new Vector2(-40, 0), new Vector2(40, 0));
+            var horizontalFixture = new Fixture(ground, horizontal)
+            {
+                CollisionLayer = (int) CollisionGroup.Impassable,
+                CollisionMask = (int) CollisionGroup.Impassable,
+                Hard = true
+            };
+
+            ground.AddFixture(horizontalFixture);
+
+            // Setup boxes
+            float a = 0.5f;
+            PolygonShape shape = new();
+            shape.SetAsBox(a, a);
+
+            var x = new Vector2(-7.0f, 0.75f);
+            Vector2 y;
+            Vector2 deltaX = new Vector2(0.5625f, 1.25f);
+            Vector2 deltaY = new Vector2(1.125f, 0.0f);
+
+            for (var i = 0; i < count; ++i)
+            {
+                y = x;
+
+                for (var j = i; j < count; ++j)
+                {
+                    var box = entityManager.SpawnEntity("BlankEntity", new MapCoordinates(0, 0, mapId)).AddComponent<PhysicsComponent>();
```

Comment:
You can pass null for the prototype to get a blank entity, we don't need to have a blank entity prototype.

Reply:
... I gotta make a robust commit real quick